### DISCUSSION TITLE
Update Grafana ?

### DISCUSSION THREAD

**Discussion by Julien4135** [tillsteinbach/VWsFriend#142] [Ideas]
Hi, I hope you don't mind too much using VWsFriend, I have a lot of questions now?

I saw that it was possible to update Grafana on the official site:

https://grafana.com/grafana/download?platform=linux&utm_source=grafana_footer

Can you tell me if it's possible suddenly and also how I have to do to have it please?

**Comment by tillsteinbach** [comment]
If you want to update Grafana you have to build a new image containing the dashboards. But I update Grafana also when releasing VWsFriend. So no need for manual updates.

**Reply by Julien4135** [reply]
Ok Thank you !
